src/pq_flash_index.cpp

```diff
@@ -86,8 +86,8 @@ namespace {
 namespace diskann {
   template<>
   PQFlashIndex<_u8>::PQFlashIndex(
-      std::shared_ptr<AlignedFileReader> &fileReader, diskann::Metric metric)
-      : reader(fileReader), metric(metric) {
```

Comment:
This seems like a fairly old version of the code. Currently there is only one PQFlashIndex constructor and the dist_cmp function determination is done elsewhere. And can we not use std::move() because upstream apps might want to retain ownership, say in the DLVS case.

Reply:
are you planning to resolve this in your newer merges?